themes/default/content/docs/concepts/options/transforms.md

```diff
@@ -0,0 +1,329 @@
+---
+title_tag: "transforms | Resource Options"
+meta_desc: The transforms resource option provides a list of transforms to apply to a resource and all of its children.
+title: "transforms"
+h1: "Resource option: transforms"
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  concepts:
+    identifier: transforms
+    parent: options
+    weight: 15
+aliases:
+- /docs/intro/concepts/resources/options/transforms/
+---
+
+The `transforms` resource option provides a list of transforms to apply to a resource and all of its children. This option is used to override or modify the inputs to the child resources of a component resource. One example is to use the option to add other resource options (such as `ignoreChanges` or `protect`). Another example is to modify an input property (such as adding to tags or changing a property that is not directly configurable).
+
+Each transform is a callback that gets invoked by the Pulumi engine. It receives the resource type, name, input properties, and resource options. The callback returns a new set of resource input properties and resource options that will be used to construct the resource instead of the original values.
+
+## Transforms vs Transformations
+
+The `transforms` option is the replacement system to [transformations](/docs/concepts/options/transformations/). Calling transform functions is managed by the engine, rather than by each SDK and so transforms apply to resources created inside remote component resources (such as those in [AWSX](https://www.pulumi.com/registry/packages/awsx/)).
```

Comment:
Maybe a sentence that compared to Transformations this works with MLCs would be helpful here?